src/Microsoft.TemplateEngine.Edge/Settings/TemplateInfoReaders/TemplateInfoReaderVersion1_0_0_0.cs

```diff
@@ -98,10 +103,13 @@ protected virtual void ReadShortNameInfo(JObject jObject, TemplateInfo info)
         protected virtual ICacheTag ReadOneTag(JProperty item)
         {
             Dictionary<string, ParameterChoice> choicesAndDescriptions = new Dictionary<string, ParameterChoice>(StringComparer.OrdinalIgnoreCase);
-            JObject cdToken = item.Value.Get<JObject>("ChoicesAndDescriptions");
-            foreach (JProperty cdPair in cdToken.Properties())
+            JObject? cdToken = item.Value.Get<JObject>("ChoicesAndDescriptions");
+            if (cdToken != null)
             {
-                choicesAndDescriptions.Add(cdPair.Name.ToString(), new ParameterChoice(null, cdPair.Value.ToString()));
+                foreach (JProperty cdPair in cdToken.Properties())
```

Comment:
Can `Properties()` be null?

Reply:
After looking at https://github.com/JamesNK/Newtonsoft.Json/blob/master/Src/Newtonsoft.Json/Linq/JObject.cs#L64 I would say no...